storage/database/s3filedb.go

```diff
@@ -76,6 +76,8 @@ func newS3FileDB(region, endpoint, bucketName string) (*s3FileDB, error) {
 	}
 
 	if !exist {
+		localLogger.Warn("creating a S3 bucket. You will be CHARGED until the bucket is deleted",
```

Comment:
How about putting `bucketName` in `localLogger`?

Reply:
I noticed that `bucketName` is already put in `localLogger` and removed duplicated things.
Thanks!